Meadowlark-js/services/meadowlark-fastify/test/Delete.test.ts

```diff
@@ -0,0 +1,45 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+import type { FastifyInstance, InjectOptions } from 'fastify';
+import * as MeadowlarkCore from '@edfi/meadowlark-core';
+import { buildService } from '../src/Service';
+
+jest.setTimeout(40000);
+
+describe('given a DELETE of a school by id', () => {
+  const schoolDeleteByIdRequest: InjectOptions = {
+    method: 'DELETE',
+    url: '/v3.3b/ed-fi/schools/1',
+    headers: { authorization: 'bearer 1234', 'content-type': 'application/json' },
+  };
+
+  let mockUpsert: any;
+  let service: FastifyInstance;
+
+  beforeAll(async () => {
+    mockUpsert = jest.spyOn(MeadowlarkCore, 'deleteIt');
+    service = buildService();
+    await service.ready();
+
+    // Act
+    await service.inject(schoolDeleteByIdRequest);
```

Comment:
This is cool. For testing Fastify, you can "inject" a request in-porcess rather than starting a server.